src/status_im/ui/screens/events.cljs

```diff
@@ -403,3 +403,10 @@
   :request-permissions
   (fn [_ [_ permissions then else]]
     {::request-permissions-fx [permissions then else]}))
+
+(handlers/register-handler-db
+  :set-swipe-position
+  [re-frame/trim-v]
+  (fn [db [item-id value]]
+    (-> db
+      (assoc-in [:chat-animations item-id :delete-swiped] value))))
```

Comment:
`->` is extra here , could be just `(assoc-in db [:chat-animations item-id :delete-swiped] value))))`

Reply:
right! updated!